src/App.vue

```diff
@@ -11,7 +11,8 @@
 import { IonApp, IonRouterOutlet, IonSplitPane } from '@ionic/vue';
 import { defineComponent } from 'vue';
 import Menu from '@/components/Menu.vue';
-import { loadingController } from '@ionic/vue';
+import { loadingController, alertController } from '@ionic/vue';
+import { useStore } from "./store";
```

Comment:
I think we need to import `useStore` from `vuex`, as importing it from `./store` might result in some warning/error.

Reply:
Made the requested change.